recipes/example/bld.bat

```diff
@@ -1,2 +0,0 @@
-@echo print('An example') > %SP_DIR%\example.py
-if errorlevel 1 exit 1
```

Comment:
Did you accidentally removed the example folder? (And I thought I would be the one doing that one day :wink:)